bsp_diff/celadon_ivi/kernel/linux-intel-lts2021/0002-Add-support-for-modifier-in-virtio-gpu.patch

```diff
@@ -0,0 +1,103 @@
+From 86419cc9db2ca251612d3b9841c3f8ad54da1238 Mon Sep 17 00:00:00 2001
+From: hangliu1 <hang1.liu@linux.intel.com>
+Date: Tue, 30 May 2023 23:13:25 -0400
+Subject: [PATCH 1/2] Add support for modifier in virtio gpu
+
+Signed-off-by: hangliu1 <hang1.liu@linux.intel.com>
+---
+ drivers/gpu/drm/drm_modeset_helper.c     |  1 +
+ drivers/gpu/drm/virtio/virtgpu_display.c |  2 +-
+ drivers/gpu/drm/virtio/virtgpu_plane.c   | 26 +++++++++++++++++++++++-
+ 3 files changed, 27 insertions(+), 2 deletions(-)
+
+diff --git a/drivers/gpu/drm/drm_modeset_helper.c b/drivers/gpu/drm/drm_modeset_helper.c
+index f858dfedf2cf..279bdb902798 100644
+--- a/drivers/gpu/drm/drm_modeset_helper.c
++++ b/drivers/gpu/drm/drm_modeset_helper.c
+@@ -93,6 +93,7 @@ void drm_helper_mode_fill_fb_struct(struct drm_device *dev,
+ 	}
+ 	fb->modifier = mode_cmd->modifier[0];
+ 	fb->flags = mode_cmd->flags;
++	printk("drm_helper_mode_fill_fb_struct : modifier = %llx in virtio_gpu_user_framebuffer_create ",fb->modifier);
```

Comment:
should not use printk, if need add log, please use DRM_DEBUG_***

Reply:
Hang said it is only for debug, could remove it.